content(cms): create Resource "using-digital-archives-for-geographical-and-archaeological-research" (#24)
stefanprobst/campus: content/resources/using-digital-archives-for-geographical-and-archaeological-research.mdx

---
title: Using Digital Archives for Geographical and Archaeological Research
lang: en
date: 2021-06-09T00:00:00.000Z
version: 1.0.0
authors:
  - digital-repository-of-ireland
contributors:
  - mccormack-barbara
  - moore-jennifer
  - swan-rónán
  - long-kevin
tags:
  - e-heritage
  - open-access
  - spatial-humanities
  - digital-archives
categories:
  - dariah
abstract: This video recording is of 'Using Digital Archives for Geographical
  and Archaeological Research', the second webinar in a three-part public
  lecture series hosted by the Digital Repository of Ireland (DRI), aimed at
  early career researchers. The webinar showcases the rich research resources
  contained in digital archival collections that can be used to advance
  geographical and archaeological research.
domain: Social Sciences and Humanities
targetGroup: Domain researchers

> type: training-module

Reviewer: video or webinar-recording?

Reviewer: ```suggestion
type: webinar-recording
```

licence: ccby-4.0
remote:
  date: 2021-03-23T00:00:00.000Z
  publisher: Digital Repository of Ireland
  url: https://doi.org/10.7486/DRI.w663hs16k
toc: false
uuid: H0ucIPCorqOlAPrrcVZBQ
---

> This video is of ‘Using Digital Archives for Geographical and Archaeological Research’, the **second** webinar in a three-part public lecture series on using digital archives for academic research hosted by the Digital Repository of Ireland (DRI) and aimed primarily at early career researchers.

Reviewer: ```suggestion
This video is of "Using Digital Archives for Geographical and Archaeological Research", the **second** webinar in a three-part public lecture series on using digital archives for academic research hosted by the Digital Repository of Ireland (DRI) and aimed primarily at early career researchers.
```

Access to brick-and-mortar archives has been limited due to COVID-19 restrictions, creating challenges for humanities researchers that rely on archival materials for their research. This webinar aims to showcase some of the rich research resources contained in digital archival collections.

Featured digital collections in this webinar include:

> - Gabriel Beranger’s ‘Rambles through the County of Dublin and some of the neighbouring ones’ and ‘Rambles thro’ the County of Dublin and some others in Ireland’: These two albums of watercolours provide illustrations of scenery and ancient monuments, general views, and details of castles and churches in Ireland. The watercolours are copies of originals (since lost) painted on expeditions undertaken by artist Gabriel Beranger, ca. 1729–1817.

Reviewer: ```suggestion
- Gabriel Beranger's "Rambles through the County of Dublin and some of the neighbouring ones" and "Rambles thro' the County of Dublin and some others in Ireland": These two albums of watercolours provide illustrations of scenery and ancient monuments, general views, and details of castles and churches in Ireland. The watercolours are copies of originals (since lost) painted on expeditions undertaken by artist Gabriel Beranger, ca. 1729–1817.
```

- Transport Infrastructure Ireland (TII) Digital Collections: The TII Digital Heritage Collections (with ca. 2,800 records) provide access to data and results arising from the archaeological and heritage work undertaken by TII over the past twenty years in the planning, development and construction of national road and light rail projects.

- Irish Historic Towns Atlas (IHTA) Online: IHTA Online forms part of IHTA Digital and has the first 28 IHTAs available to search freely online as downloadable pdfs. They are grouped thematically by town origin (e.g., monastic, Viking, Anglo-Norman, early modern, Gaelic and plantation, eighteenth-century and nineteenth-century towns).

## Learning Outcomes

After watching this webinar, early career researchers will be familiar with some of the digital archival collections that can be explored to advance geographical and archaeological research, they will have been introduced to many helpful online research resources, and offered a starting point from which to develop research methodologies for conducting digital archival research.

> <CTA

Reviewer: ```suggestion
<ExternalResource
```
